### ISSUE TITLE
Error with 3.0.0

### ISSUE THREAD

**Issue by flauc** [ngrx/store-log-monitor#4]
I've just updated to 3.0.0 and this is the error i'm getting 

```
SyntaxError: Unexpected token < 
at Object.eval (http://localhost:2000/node_modules/@ngrx/store-log-monitor/index.js:4:22)
```

It has something to do with me using system.js, but everything worked perfectly in an earlier release. 

P.S 

This is the line in index.js:

```
var dock_monitor_1 = require('./dock-monitor');
```

**Comment by Salimlou** [comment]
I have the same issue with systemjs builder.

It seems that he doesn't look for index.js inside docker-monitor folder

**Comment by flauc** [comment]
I've tried to add `/index` in to every require that misses it and then everything works fine. It's a temporary fix but it works.

**Comment by karlhaas** [comment]
I have the same issue: 

```
localhost/:142 Error: (SystemJS) XHR error (404 Not Found) loading http://localhost:5555/node_modules/@ngrx/store-log-monitor/dock-monitor.js
```

I'm using an older version of angular2seed with the system js configuration `this.SYSTEM_CONFIG_DEV.packageConfigPaths.push(`${this.APP_BASE}node_modules/@ngrx/*/package.json`);` for ngrx.

Building the dist bundle has the same error:

```
[11:45:53] Starting 'build.bundles.app'...
Unhandled rejection Error on fetch for @ngrx/store-log-monitor/dock-monitor.js at file:///home/karl/Work/cnc/feedbackr/Frontend/node_modules/@ngrx/store-log-monitor/dock-monitor.js
    Loading @ngrx/store-log-monitor/index.js
    Loading dist/tmp/app/app.module.js
    Loading dist/tmp/app/main.js
    Error: ENOENT: no such file or directory, open '/home/karl/Work/cnc/feedbackr/Frontend/node_modules/@ngrx/store-log-monitor/dock-monitor.js'
    at Error (native)
```

**Comment by attilacsanyi** [comment]
Same issue for me, with webpack2 angular-cli working well, but not with systemjs.

**Comment by SamVerschueren** [comment]
Same here. SystemJS does not pick up the `index.js` file when pointing to a directory, instead it looks for the JS file with that name (e.g. `dock-monitor` -> `dock-monitor.js`).

**Comment by aaweb** [comment]
Same here, trying to add it to angular2-seed-advanced, it's trying to load '@ngrx/store-log-monitor/dock-monitor.js' and there doesn't seem to be an easy way to work around.

**Comment by brandonroberts** [comment]
I'm working on a PR to fix this issue

**Comment by attilacsanyi** [comment]
Any estimation when is going to be in master @brandonroberts ? Thanks in advance.

**Comment by brandonroberts** [comment]
@attilacsanyi the PR is in.

**Comment by attilacsanyi** [comment]
It was really fast @brandonroberts really appreciate 😉

**Comment by MikeRyanDev** [comment]
Fixed in v3.0.2
